packages/@aws-cdk/aws-secretsmanager/README.md

```diff
@@ -39,6 +39,30 @@ const secret = secretsmanager.Secret.fromSecretAttributes(scope, 'ImportedSecret
 SecretsManager secret values can only be used in select set of properties. For the
 list of properties, see [the CloudFormation Dynamic References documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html).
 
+### Grant permission to use the secret to a role
+
+You must grant permission to a resource for that resource to be allowed to 
+use a secret. This can be achieved with the `Secret.grantRead` and/or 
+`Secret.grantWrite` method, depending on your need:
+
+```ts
+const role = new iam.Role(stack, 'SomeRole', { assumedBy: new iam.AccountRootPrincipal() });
+const secret = new secretsmanager.Secret(stack, 'Secret', {});
```

Comment:
You can leave out the `props` argument, it's optional for `Secret`. So, just `new secretsmanager.Secret(stack, 'Secret');`.

Reply:
fixed